modules/social_features/social_branding/src/Plugin/GraphQL/SchemaExtension/AppSchemaExtension.php

```diff
@@ -0,0 +1,143 @@
+        ->map('brandingColors', $builder->fromParent())
+        ->map('paletteName', $builder->fromValue('brand-accent-text'))
+        ->map('configName', $builder->fromValue('accent_text'))
+    );
+    $registry->addFieldResolver('PlatformBrandColorScheme', 'link',
+      $builder->produce('platform_branding_colors_load_color_by_name')
+        ->map('brandingColors', $builder->fromParent())
+        ->map('paletteName', $builder->fromValue('brand-link'))
+        ->map('configName', $builder->fromValue('link'))
+    );
+    $registry->addFieldResolver('PlatformBrandColorScheme', 'navbarBackground',
+      $builder->produce('platform_branding_colors_load_color_by_name')
+        ->map('brandingColors', $builder->fromParent())
+        ->map('paletteName', $builder->fromValue('navbar-bg'))
+        ->map('configName', $builder->fromValue('navbar_bg'))
+    );
+    $registry->addFieldResolver('PlatformBrandColorScheme', 'navbarText',
+      $builder->produce('platform_branding_colors_load_color_by_name')
+        ->map('brandingColors', $builder->fromParent())
+        ->map('paletteName', $builder->fromValue('navbar-text'))
+        ->map('configName', $builder->fromValue('navbar_text'))
+    );
+    $registry->addFieldResolver('PlatformBrandColorScheme', 'navbarActiveBackground',
+      $builder->produce('platform_branding_colors_load_color_by_name')
+        ->map('brandingColors', $builder->fromParent())
+        ->map('paletteName', $builder->fromValue('navbar-active-bg'))
+        ->map('configName', $builder->fromValue("navbar_active_bg'"))
+    );
+    $registry->addFieldResolver('PlatformBrandColorScheme', 'navbarActiveText',
+      $builder->produce('platform_branding_colors_load_color_by_name')
+        ->map('brandingColors', $builder->fromParent())
+        ->map('paletteName', $builder->fromValue('navbar-active-text'))
+        ->map('configName', $builder->fromValue("navbar_active_text'"))
+    );
+    $registry->addFieldResolver('PlatformBrandColorScheme', 'navbarSecondaryBackground',
+      $builder->produce('platform_branding_colors_load_color_by_name')
+        ->map('brandingColors', $builder->fromParent())
+        ->map('paletteName', $builder->fromValue('navbar-sec-bg'))
+        ->map('configName', $builder->fromValue("navbar_sec_bg'"))
+    );
+    $registry->addFieldResolver('PlatformBrandColorScheme', 'navbarSecondaryText',
+      $builder->produce('platform_branding_colors_load_color_by_name')
+        ->map('brandingColors', $builder->fromParent())
+        ->map('paletteName', $builder->fromValue('navbar-sec-text'))
+        ->map('configName', $builder->fromValue("navbar_sec_text'"))
+    );
+
+    // Color fields.
+    $registry->addFieldResolver('Color', 'hexRGB',
+      $builder->produce('color_hex')
+        ->map('color', $builder->fromParent())
+    );
+    $registry->addFieldResolver('Color', 'rgba',
+      $builder->produce('color_rgba')
+        ->map('color', $builder->fromParent())
```

Comment:
```suggestion
      $builder->fromParent()
```

Since the data producer is just passthrough, we can also do that here :)